tests/test_set_grpc_max_size_in_bytes.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2022 Ant Group Co., Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import multiprocessing
+import pytest
+import fed
+
+@fed.remote
+class My:
+    def __init__(self) -> None:
+        self._val = ""
+
+    def set_val(self, val):
+        self._val += val
+        return self._val
+
+
+@fed.remote
+def concat_str(val1, val2):
+    return f"{val1} {val2}"
+
+def run(party):
+    cluster = {
+        'alice': {'address': '127.0.0.1:11010'},
+        'bob': {'address': '127.0.0.1:11011'},
+    }
+    fed.init(address='local', cluster=cluster, party=party, cross_silo_messages_max_size_in_bytes=10)
```

Comment:
In the meantime, I try a very small value for `cross_silo_messages_max_size_in_bytes`.